FrontEnd/src/components/Contact/Contact.module.css

```diff
@@ -72,27 +92,211 @@
+    resize: none;
+    outline: none;;
+}
+
+.contact__textarea,
+.contact__select {
+    width: 400px;
+    padding: 10px;
+    border: 1px solid var(--border-color);
+    border-radius: 4px;
+    border: 1px solid var(--border-color);
+    transition: border-color 0.3s ease;
+    box-sizing: border-box;
+    min-height: 40px;
+    max-height: 60px;
+    resize: none;
+}
+
+.contact__textarea {
+    resize: vertical;
+    min-height: 120px;
+}
+
+.contact__input::placeholder {
+    opacity: 0.3; /* Повністю видимий плейсхолдер */
+}
+.contact__input:focus,
+.contact__textarea:focus,
+.contact__select:focus {
+    border-color: #D9D9D9;
+}
+
+.contact__select.placeholder {
+    color: gray;
+}
+
+.contact__button_send {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #1F9A7C;
+    box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.04);
+    color: white;
+    font-feature-settings: 'calt' off;
+    font-style: normal;
+    border-radius: 4px;
+    border: 1px;
+    gap: 10px;
+    font-family: var(--font-main);
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: -0.01em;
+    text-align: center;
+    cursor: pointer;
+    width: 100px;
+    height: 32px;
+}
+
+.contact__button_send:hover {
+    background-color: #218838;
+}
+
+.contact__button_cancel {
+    display: flex;
+    background-color: #e74c3c;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.04);
+    color: white;
+    font-feature-settings: 'calt' off;
+    font-style: normal;
+    border-radius: 4px;
+    border: 1px;
+    gap: 10px;
+    font-family: var(--font-main);
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: -0.01em;
+    text-align: center;
+    cursor: pointer;
+    width: 100px;
+    height: 32px;
+
+}
+
+.contact__button_cancel:hover {
+    background-color: #c82333;
+}
+
+.contact__error {
+    display: flex;
+    align-items: flex-start;
+    gap: 10px;
+    align-self: stretch;
+    color: var(--red-red-100, #f34444);
+    font-family: var(--font-messages);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 16px;
+}
+
+.contact__button_container {
+    display: flex;
+    gap: 10px;
+    justify-content: flex-start;
+    margin-top: 20px;
+}
+
+.contact__button_modal_container {
+    display: flex;
+    gap: 10px;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20px;
+
+}
+
+.modal_feedback {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgba(0, 0, 0, 0.7);
+    z-index: 1000;
+
+}
+
+.modal_feedback_content {
+    background-color: white;
+    padding: 20px;
+    position: relative;
+    width: 550px;
+    height: 150px;
+    text-align: center;
+    display: flex; /* Включаємо Flexbox */
```

Comment:
I'm not sure these comments are appropriate.

Reply:
Done.